src/removeLastHangulCharacter.ts

```diff
@@ -0,0 +1,42 @@
+import { combineHangulCharacter } from './combineHangulCharacter';
+import { disassembleHangulToGroups } from './disassemble';
+import { excludeLastElement } from './_internal';
+
+/**
+ * @name removeLastHangulCharacter
+ * @description
+ * 인자로 주어진 한글 문자열에서 가장 마지막 문자 하나를 제거하여 반환합니다.
+ * ```typescript
+ * removeLastHangulCharacter(
+ *   // 한글 문자열
+ *   words: string
+ * ): string
+ * ```
+ * @example
+ * removeLastHangulCharacter('안녕하세요 값') // 안녕하세요 갑
+ * removeLastHangulCharacter('프론트엔드') // 프론트엔ㄷ
+ * removeLastHangulCharacter('일요일') // 일요이
+ */
+export function removeLastHangulCharacter(words: string) {
```

Comment:
한 단어를 받는 인자라면 복수인 words가 아닌, word로 해도 좋을 것 같아요! 

```suggestion
export function removeLastHangulCharacter(words: string) {
```

Reply:
"안녕하세요 값"처럼 2개 이상의 word도 받을 수 있어서 복수형으로 정의했습니다!

Reply:
오오 맞네요! 좋아요~!